src/workloads/scale/CreateDropView.yml

```diff
@@ -0,0 +1,81 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload loads a collection with documents and then repeatedly
+  create and drops a view on that collection.
+Keywords:
+- view
+- create
+- drop
+
+dbname: &dbname viewDB
+DocumentCount: &NumDocs 100000
+CollectionCount: &NumColls 1
+
+
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 100
+    Threads: 1
+    DocumentCount: *NumDocs
+    Database: *dbname
+    CollectionCount: *NumColls
+    Document: &doc
+      id: {^RandomInt: {min: 0, max: *NumDocs}}
+      a: {^RandomInt: {min: 0, max: *NumDocs}}
+      c: &string {^RandomString: {length: 50}}  # Adjust this so the doc comes out as 100 B.
+    Indexes:
+    - keys: {id: 1}
+    - keys: {a: 1}
+  - Phase: 1..3
+    Nop: true
+
+- Name: QuiesceBetweenLevels
```

Comment:
We have a quiesce actor for this. See: https://github.com/mongodb/genny/blob/48b3fac2e0c8c29b5f196e1263acc8d89e3762ed/src/workloads/docs/QuiesceActor.yml. Please use that instead of the explicit FsycnCommand.

Reply:
Oh nice - replaced!

Reply:
FYI, Jim had I think a good suggestion of creating a ticket for replacing the instances where RunCommand fsyncs are used with the QuieseActor. Created [PERF-3545](https://jira.mongodb.org/browse/PERF-3545)